core/src/mcp.test.ts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+import { mcpWrapper } from "./mcp";
+
+import { expect, test } from "vitest";
+
+test("aws-mcp-test", async () => {
+  console.log("Initializing MCP wrapper...");
+
+  // Initialize the wrapper with the configuration
+  await mcpWrapper.initialize({
+    servers: [
+      {
+        type: "stdio",
+        command: "uvx",
+        args: [
+          "--from",
+          "awslabs-aws-documentation-mcp-server",
+          "awslabs.aws-documentation-mcp-server.exe",
+        ],
```

Comment:
This is why it'd be much better to have a dedicated in tree (our own source code) mock server, as this is tied to the windows environment and also just takes a long time to download.